frontend/src/pages/GraphicalSelector/CourseInfo/GraphicalCourseTag.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { MouseEventHandler } from 'react';
+import S from './styles';
+
+type Props = {
+  name: string;
+  onClick?: MouseEventHandler<HTMLSpanElement>;
+};
+
+const CourseTag = ({ name, onClick }: Props) => (
```

Comment:
Dont we have a course tag component that we can reuse?

Reply:
No, not one that I can pass a callback too 😞